### PR TITLE
ENCD-4170-exclude-restricted-files

### PR DESCRIPTION
**Pull request by zoldello**

### CODE REVIEW

File: src/encoded/batch_download.py [hunk 1]
```diff
@@ -84,6 +84,7 @@ def includeme(config):
     ('Platform', ['files.platform.title']),
```
Comment (zoldello) [inline src/encoded/batch_download.py]: @caseylitton  What code change is still required?

Comment (caseylitton) [reply src/encoded/batch_download.py]: I suggest slacking people when you need to remind them about code review

File: src/encoded/batch_download.py [hunk 2]
```diff
@@ -523,6 +531,8 @@ def restricted_files_present(exp_file):
         return True
     return False
 
+def is_no_file_available(exp_file):
```
Comment (caseylitton) [inline src/encoded/batch_download.py]: Two spaces before and after top level functions.

File: src/encoded/batch_download.py [hunk 3]
```diff
@@ -130,9 +131,12 @@ def includeme(config):
     ('Derived from', ['files.derived_from']),
     ('S3 URL', ['files.cloud_metadata']),
     ('Size', ['files.file_size']),
+    ('No File Available', ['file.no_file_available']),
     ('Restricted', ['files.restricted'])
 ])
 
+_excluded_columns = ('Restricted', 'No File Available')
+
```
Comment (caseylitton) [inline src/encoded/batch_download.py]: top space between top level vars and functions.

File: src/encoded/batch_download.py (hunk 1 above)
Comment (zoldello) [reply src/encoded/batch_download.py]: @caseylitton Acknowledged

**Pull request merged**